main.py

```diff
@@ -56,6 +51,7 @@
             # This feature will stop buying upgrades when the balance is less than the price of the best card.
             "wait_for_best_card": False,  # Recommended to keep it True for high level accounts
             "auto_get_task": True,  # Enable auto get (Youtube/Twitter and ...) task to True, or set it to False to disable
+            "parallel_upgrades_max_price_per_hour": 6_000_000,  # Enable parallel card buy if 1k of profit costs less than value
```

Comment:
Need to add a better description with more detail and also support 0 for disabling the feature.
Also will be good, if you allow them to limit loops, like 2-3 or 5 or etc ...

Reply:
added a flag to disable-enable feature.

I don't think it is a good idea to limit loops. I do not see benefits in such a behavior